### PR TITLE
Plugin property common action for key value pairs

### PR DESCRIPTION
**Pull request by sawantpritam**
@itsankit-google @rmstar Please review.
This PR contains :
1. Common action to enter key value pairs.

image

2. Common action to change datatype of fields in output schema

image

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -852,4 +869,33 @@ public static void verifyRadioButtonPluginPropertySelectedValue(String pluginPro
...
+    }
+    Map<String, String> properties =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonKeyValuePairs));
+    int index = 0;
+    for (Map.Entry<String, String> entry : properties.entrySet()) {
+      ElementHelper.sendKeys(CdfPluginPropertiesLocators.locatePropertyKey(
+        pluginPropertyDataCyAttribute, index), entry.getKey());
+      ElementHelper.sendKeys(CdfPluginPropertiesLocators.locatePropertyValue(
+        pluginPropertyDataCyAttribute, index), entry.getValue());
+      ElementHelper.clickOnElement(CdfPluginPropertiesLocators.locatePropertyAddRowButton(
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: I think this will end up adding an extra row than required?

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Updated logic to add row.

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 2]
```diff
@@ -113,6 +113,23 @@ public static void selectMacroActionOfOutputSchemaProperty() {
...
+   *                              is the json array with field name and datatype
+   *                              ({@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE})
+   */
+  public static void changeDataTypesInOutputSchema(String jsonFieldList) {
+    Map<String, String> outputSchemaFieldList =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonFieldList));
+    for (Map.Entry<String, String> entry : outputSchemaFieldList.entrySet()) {
+      ElementHelper.clickOnElement(CdfSchemaLocators.outputSchemaDataTypeDropdown(entry.getKey()));
+      SeleniumDriver.getDriver()
+        .findElement(CdfSchemaLocators.outputSchemaDataTypeOption(entry.getKey(), entry.getValue())).click();
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Can we use `ElementHelper` here?

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: No. Element helper click method first waits for element to be clickable, then scroll to that element and then click.
Output schema datatype options list element is not scrollable. We get exception while scrolling -
`element not interactable: [object HTMLOptionElement] has no size and location`
So direct click action is used.

**Pull request merged**